File: visual-model-ui/src/main/java/org/visual/model/ui/bar/SystemMenuBar.java
package org.visual.model.ui.bar;

import javafx.scene.control.Menu;
import javafx.scene.control.MenuBar;
import javafx.scene.control.MenuItem;
import lombok.val;

public class SystemMenuBar extends MenuBar {

    {
        this.setUseSystemMenuBar(true);
        val menu = new Menu("test");
        val it = new MenuItem();
        it.setText("test");
        menu.getItems().add(it);
        getMenus().add(menu);
    }
}

File: visual-model-ui/src/main/kotlin/org/visual/model/ui/bar/SystemMenuBar.kt
package org.visual.model.ui.bar

import javafx.scene.control.Menu
import javafx.scene.control.MenuBar
import javafx.scene.control.MenuItem

class SystemMenuBar : MenuBar() {
    init {
        this.isUseSystemMenuBar = true
        val menu = Menu("test")
        val it = MenuItem()
        it.text = "test"
        menu.items.add(it)
        menus.add(menu)
    }
}
